src/sections/community/styles.scss

```diff
@@ -194,6 +338,103 @@
       }
     }
   }
+
+  &__navigator {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-left: auto;
+    margin-right: auto;
+    
+    // To help move the navigator all the way to the left, outside of normal max-width of parent
+    // 960px is the max width of the div above the <main> tag
+    @media (min-width: 960px) {
+      margin-left: calc(-1 * ((80vw - 960px) / 2));
+    }
+    
+    align-items: center;
+    width: 80vw;
+  }
+
+  &__image-grid {
+    display: grid;
+    height: 350px;
+    width: 90%;
+    gap: 0.1rem;
```

Comment:
I think having a larger gap might look nicer

This is 0.5rem:

![image](https://github.com/user-attachments/assets/96a3c2d1-c08d-42b7-86ea-6fbf3b3c9b22)